envoy/http/alternate_protocols_cache.h

```diff
@@ -99,6 +99,15 @@ class AlternateProtocolsCache {
   virtual void setAlternatives(const Origin& origin,
                                std::vector<AlternateProtocol>& protocols) PURE;
 
+  /**
+   * Sets the bandwidth and rtt estimates for an endpoint, assuming the endpoint
+   * exists in the cache.
+   * @param origin The origin to set network characteristics for.
+   * @param srtt The smothed round trip time for the origin.
+   * @param bytes_per_second the estimated bandwidth in bytes per second.
+   */
+  virtual void setRttBandwidth(const Origin& origin, int64_t srtt, int64_t bytes_per_second) PURE;
```

Comment:
Instead of int for srtt, can we use std::chrono::milliseconds to make the units clear?

Reply:
done here and elsewhere